src/metrics/mapping.pl

```diff
@@ -684,7 +684,7 @@
     0x33 => 0x220B,         # \ni
     0x34 => 0x25B3,         # \bigtriangleup and \triangle
     0x35 => 0x25BD,         # \bigtriangledown
-    0x36 => [0x338,-894,0], # \not (combining)
+    0x36 => 0x338,          # \not (combining)
```

Comment:
Same question for these comments.

Reply:
I'll copy the comments from makeFF here as well.